kyo-tag/shared/src/main/scala/kyo/tags.scala

```diff
@@ -14,24 +14,98 @@ object Tag:
 
     extension [T](t1: Tag[T])
 
-        def show =
+        def show: String =
             val decoded = t1.tpe.drop(2).takeWhile(_ != ';')
             fromCompact.getOrElse(decoded, decoded)
 
-        def <:<[U](t2: Tag[U]): Boolean =
+        infix def <:<[U](t2: Tag[U]): Boolean =
```

Comment:
The compiler seems quite finicky when it requires the infix keyword...

Reply:
yeah, it started complaining about it out of the blue. I imagine it could be a bad interaction with extension methods.